src/python/pants/releases/packages.py

```diff
@@ -101,7 +102,7 @@ def check_ownership(users):
   users = set(user.lower() for user in users)
   non_expected_users = users.difference(expected_package_owners)
   if non_expected_users:
-    raise ValueError('{} are not expected releasers. You may want to get added, as per https://www.pantsbuild.org/release.html#owners and then add yourself to expected_package_owners in file build-support/bin/release.py'.format(", ".join(non_expected_users)))
+    raise ValueError('{} are not expected releasers. You may want to get added, as per https://www.pantsbuild.org/release.html#owners and then add yourself to expected_package_owners in file src/python/pants/releases/packages.py'.format(", ".join(non_expected_users)))
```

Comment:
Hm. Given where it runs, this check seems more annoying than helpful.

Reply:
I thought so too. happy to eliminate the check altogether if you think that makes more sense.

Reply:
From looking at it, it's not clear why there is a literal set, given that https://github.com/twitter/pants/blob/18da8cfdd5e715f5b4deeecb046f7284fab4e061/src/python/pants/releases/packages.py#L109 checks ownership using actual live values.